docs/src/pages/ui/components/authenticator/index.page.mdx

```diff
@@ -385,3 +385,28 @@ The example below uses a `<style>` tag to change the default colors to a dark th
     with an Amplify backend.
   </p>
 </Example>
+
+### Additional CSS Styling
+
+You can also override the authenticator's `amplify-*` classes.
+
+For example, if you'd like to hide the sign up tab, you can override the `amplify-tabs` class.
+
+```css
+.amplify-tabs {
+  display: none;
+}
+```
```

Comment:
This is clever! I think we can keep this until we need a first class support for this.

Reply:
👍